<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TWR_Form</class>
 <widget class="QWidget" name="TWR_Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>620</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TWR_v1.0.0</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>基于TWR的红外内部缺陷智能检测系统</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item alignment="Qt::AlignRight">
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>温度数据文件：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_fileName">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="placeholderText">
          <string>请输入文件路径</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_loadFile">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item alignment="Qt::AlignHCenter|Qt::AlignTop">
          <widget class="QLabel" name="label_3">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>宋体</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>原始红外图像</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter|Qt::AlignTop">
          <widget class="QLabel" name="label_ori">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>280</width>
             <height>280</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>280</width>
             <height>280</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::WinPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="pushButton_showPic">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>120</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>宋体</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>加载图像</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_drawROI">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>120</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>宋体</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>绘制ROI</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
          <widget class="QTabWidget" name="tabWidget">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>310</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>312</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>宋体</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="elideMode">
            <enum>Qt::ElideNone</enum>
           </property>
           <widget class="QWidget" name="tab_lag">
            <attribute name="title">
             <string>时滞图</string>
            </attribute>
            <widget class="QLabel" name="label_lag">
             <property name="geometry">
              <rect>
               <x>7</x>
               <y>2</y>
               <width>280</width>
               <height>280</height>
              </rect>
             </property>
             <property name="minimumSize">
              <size>
               <width>280</width>
               <height>280</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>280</width>
               <height>280</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Sunken</enum>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </widget>
           <widget class="QWidget" name="tab_pha">
            <attribute name="title">
             <string>相位图</string>
            </attribute>
            <widget class="QLabel" name="label_pha">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="geometry">
              <rect>
               <x>7</x>
               <y>2</y>
               <width>280</width>
               <height>280</height>
              </rect>
             </property>
             <property name="minimumSize">
              <size>
               <width>280</width>
               <height>280</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>280</width>
               <height>280</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Sunken</enum>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </widget>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="pushButton_featureExtract">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>120</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>宋体</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>提取特征</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_detect">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>120</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>宋体</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>检测缺陷</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label_5">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>宋体</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>检测结果</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label_detect">
           <property name="minimumSize">
            <size>
             <width>280</width>
             <height>280</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>300</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>宋体</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>缺陷个数：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_numDefects">
             <property name="minimumSize">
              <size>
               <width>180</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>30</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>10</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="plainTextEdit_logs">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>940</width>
         <height>125</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1920</width>
         <height>160</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="placeholderText">
        <string>欢迎使用本软件，请先导入温度数据文件。</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
